src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,121 @@
+package duke.parser;
+
+import duke.exception.DescriptionEmptyException;
+import duke.exception.DueTimeEmptyException;
+import duke.command.*;
+import duke.exception.InvalidCommandException;
+
+public class Parser {
+
+    public final int PARAM_DELIMIT_LIMIT = 2;
```

Comment:
Good job extracting this out as a constant so that it doesn't appear as a magic number.